front/app/modules/commercial/customizable_navbar/admin/components/NavbarTitleField.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Formik field for the navbar item title, similar to the fields in `PageForm/fields`.

Reply:
So similar that maybe we could use the same form ? A page can become a navbar item pretty easily, and here having the same behaviour sounds like a plus : now if I understand correctly if you create a page, it can have a very long title, then you add it to the navbar and the title is too long...

Reply:
Will try to improve this in the next PR